src/documentation/read_ntfs_fares.md

```diff
@@ -0,0 +1,32 @@
+# Reading NTFS fares
+## Introduction
+This document describes how [NTFS fares](https://github.com/CanalTP/navitia/blob/dev/documentation/ntfs/ntfs_fare_extension_fr.md) are loaded in Navitia Transit Model.
+
+In this initial version: 
+- only tickets on origin-destination stops are taken into account
+- the NTFS file *fares.csv* is ignored, as the NTM fares model does not yet support special conditions on tickets (e.g validity duration, number of transfers allowed, constraints on networks/lines) 
+- tickets specified in a currency different than EUR are ignored
+- only tickets on origin-destination stops are taken into account
+
+### Loading Tickets
+
+NTM property | NTFS file | NTFS field | Note/mapping rule
+--- | --- | --- | ---
+id | prices.csv | \*clef de ticket\* | 
```

Comment:
Is the rendering with  `*` on purpose?

Reply:
I used this rendering in order to highlight the fact that this file (prices.csv) has no header row and the NTFS fields are not actually named.